packages/components/src/calendar/style.scss

```diff
@@ -167,6 +163,13 @@
 .woocommerce-calendar__input-error {
 	display: none;
 
+	.is-error .woocommerce-calendar__input-text:focus + span & {
+		display: block;
+		left: 50% !important;
+		position: absolute;
+		top: auto !important;
+	}
```

Comment:
Using Chrome and looking at a RTL language, the `left` inline style doesn't get overwritten:

![Screen Shot 2019-03-15 at 9 11 04 AM](https://user-images.githubusercontent.com/1922453/54388316-5b5f7500-4702-11e9-9953-69bbe8737587.png)

It looks like the `left: 50% !important;` gets rewritten by the rtl plugin to `right: 50% !important;`. If I change it back to `left: 50% !important;` in the console, it works.

Reply:
You're right! I tested it in Firefox and wrongly assumed it would work in other browsers too.

I [made](https://github.com/woocommerce/woocommerce-admin/pull/1803/commits/d4847dcc5e3ecc7614cbe08f9ee05d1deeb82226) that CSS ruleset to be ignored by the RTL plugin, so it should work fine now in both browsers.